docs/integrations/amazon-aws/amazon-elastic-container-service-container-insights-cloudwatch.md

```diff
@@ -0,0 +1,643 @@
+---
+id: elastic-container-service-container-insights-cloudwatch
```

Comment:
@nmohantyctr This doc already exists. Unless your PR is making new changes on top of what's there , you can close this out https://help.sumologic.com/docs/integrations/amazon-aws/elastic-container-service-container-insights-cloudwatch/